src/test/integration/integration_tests.py

```diff
@@ -34,7 +34,11 @@ def validate_filter_param(self, res, type, param):
         valid_id = res.json()["data"][0]["attributes"][param]
         filtered_objects = utils.get(type, {param: valid_id})
         validate_response(self, filtered_objects, 200)
-        self.assertIn(res.json()["data"][0], filtered_objects.json()["data"])
+        for object in filtered_objects.json()["data"]:
+            if object["attributes"][param] != valid_id:
+                self.fail("{} object with {} of {} found. Should be {}".format(
+                    type, param, object["attributes"][param], valid_id
+                ))
```

Comment:
This change was due to an error I was getting in person with @tsoliangwu0130. I was trying to test that the same `vehicle` object was in GET /vehicles with/without its `routeID` specified as a parameter. Since I was asserting that the entire object must be identical, errors would randomly occur when fields that change rapidly such as something like `"lastUpdated"` would change some time between the two requests. I changed my validation method to compare only the `routeID` field which should not rapidly change like some of the others.